libyul/AsmAnalysis.cpp

```diff
@@ -323,7 +338,7 @@ vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 		for (size_t i = 0; i < parameterTypes->size(); ++i)
 			expectType((*parameterTypes)[i], argTypes[i], locationOf(_funCall.arguments[i]));
 
-	if (m_success)
+	if (success())
```

Comment:
Ok, this is another place, but also here, checking if we added a new error inside this routine would be the better behaviour.